palace/utils/iodata.hpp

```diff
@@ -44,6 +45,9 @@ class IoData
   // Nondimensionalize input values for use in the solver, including the mesh coordinates.
   void NondimensionalizeInputs(mfem::ParMesh &mesh);
 
+  // Return the mesh scaling factor, (Lc / L0), for mesh IO.
+  double GetMeshScaleFactor() const { return Lc / model.L0; }
```

Comment:
`GetLengthScaleFactor`? Since all units in the config file are expected to by in units of `model.L0 * m`